licenses/TEMPLATE_LICENSE.rst

```diff
@@ -0,0 +1,31 @@
+This project is based upon the Astropy package template
```

Comment:
@Cadair here also AstroPy is present. Is this a bug as well?

Reply:
SunPy's package template is based on Astropy's package template, so we do need the license for Astropy's package template in there.  I'm not sure if there's also a license for SunPy's package template.